docs/source/conf.py

```diff
@@ -158,6 +158,9 @@
     "get_model_info",
     "get_storage_info",
     "hierarchical_pickle",
+    # torch.accelerator
```

Comment:
Adding entries to this and the allowlist below is not ok. We need to keep these functions documented until they are removed.

Reply:
Updated.